2 dz/6/main.cpp

```diff
@@ -0,0 +1,57 @@
+/*Реализовать консольную игру "Быки и коровы"*/
+
+#include "numberGenerator.h"
+#include "bullsAndCows.h"
+
+void startTheGame()
+{
+    unsigned int size = 0;
+    std::cout << "enter the number of digits (no more than 10)" << std::endl;
+    std::cin >> size;
+    if (size >= 10)
+    {
+        std::cout << "Why are you gay?" << std::endl;
```

Comment:
:D
Caution, GitHub, or other people from the modern world can ban you for using this word with a negative meaning.